docs/sources/alerting/unified-alerting/_index.md

```diff
@@ -1,11 +1,55 @@
 +++
 title = "Unified Alerts"
-aliases = ["/docs/grafana/latest/alerting/rules/", "/docs/grafana/latest/alerting/metrics/"]
+aliases = ["/docs/grafana/latest/alerting/metrics/"]
 weight = 113
 +++
 
 # Unified alerts
+Alerts allow you to know about problems in your systems moments after they occur. Robust and actionable alerts help you identify and resolve issues quickly, minimizing disruption to your services.
 
+>**Note:** This information is for the new, Unified Alerts. This is an [opt-in]({{< relref"./opt-in.md" >}}) feature released in Grafana 8.0. Grafana still supports [dashboard alerts]({{< relref "../old-alerting/_index.md" >}}) out of the box
 
+Alerts have four main components:
+
+- Alert rule - One or more conditions, the frequency of evaluation, and the (optional) duration that a condition must be met before notifying.
+- Contact point - A channel for sending notifications when the conditions of an alert rule are met. (Available in unified alerting only.)
+- Notification policy - A set of matching and grouping criteria used to determine where, and how frequently, to send notifications.
+- Silences - Date and matching criteria used to silence notifications. (Available in unified alerting only.)
+
+## Alert tasks
+
+You can perform the following tasks for alerts:
+
+- [Create an alert rule]({{< relref "./create-alert-rule.md" >}})
+- [View existing alert rules and their current state]({{< relref "./view-alerts.md" >}})
+- [Test alert rules and troubleshoot]({{< relref "./troubleshoot-alerts.md" >}})
+- [Add or edit an alert contact point]({{< relref "./contact-points.md" >}})
+
```

Comment:
notification policies deserve an entry as well

Reply:
Also added silences

Reply:
The consensus we reached is that we will use "alerts" (referring to new alerts) and legacy alerts (referring to old Grafana alerts). This information should be at the top level.